### ISSUE TITLE
[COGNITION]: GIBCT - Caution flags' bulleted lists SHOULD have valid HTML for screen reader usability

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7987]
**Feedback framework**
- **⚠️ Should** fix this issue 1-3 sprints post launch
- [508 defect 3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Description

Some of the bulleted lists of caution flags have an extra `<div>` inside the list item, and it's causing screen readers to break the bullet point and its text into two `DOWN_ARROW` presses for JAWS and NVDA users. This breaks up the understanding of the bullet points and can be easily fixed by removing the extra DIV. Screenshot below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Screen readers read the bullet point and text in one group, not forcing users to press `DOWN_ARROW` or `i` twice.

## WCAG or Vendor Guidance (optional)

* [Meaningful Sequence: Understanding SC 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

## Screenshots or Trace Logs

![Screen Shot 2020-04-14 at 8 40 48 AM](https://user-images.githubusercontent.com/934879/79231514-ca3e5e80-7e2b-11ea-88f7-6ab8be868062.png)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12233

**Comment by delli-gatti-michael** [comment]
Local testing complete

**Comment by 1Copenut** [comment]
Confirmed in Staging as well. VoiceOver is reading the list items appropriately. Moving to close.
